geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/AvroSimpleFeatureUtils.scala

```diff
@@ -29,6 +29,7 @@ object AvroSimpleFeatureUtils {
 
   val FEATURE_ID_AVRO_FIELD_NAME: String = "__fid__"
   val AVRO_SIMPLE_FEATURE_VERSION: String = "__version__"
+  val AVRO_SIMPLE_FEATURE_USERDATA: String = "__userdata__"
 
   // Increment whenever encoding changes and handle in reader and writer
   val VERSION: Int = 2
```

Comment:
Do we care that the AvroDataFile version is 1 while the AvroSimpleFeatureUtils version is 2?  Is there any merit to having those numbers synchronized?